### PR TITLE
deps: Bump nmpolicy to v0.2.1

### PR DESCRIPTION
**Pull request by qinqon**
**Is this a BUG FIX or a FEATURE ?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
/kind enhancement

**What this PR does / why we need it**:
Improve capture field error messages.

**Special notes for your reviewer**:

**Release note**:

```release-note
Improve capture field error messages.
```

### CODE REVIEW

File: pkg/nmpolicy/generate.go [hunk 1]
```diff
@@ -93,13 +93,13 @@ func convertGeneratedStateToEnactmentConfig(nmpolicyGeneratedState nmpolicytypes
 	for captureKey, capturedState := range nmpolicyGeneratedState.Cache.Capture {
 		capturedState := nmstateapi.NodeNetworkConfigurationEnactmentCapturedState{
 			State: nmstateapi.State{
-				Raw: capturedState.State,
+				Raw: []byte(capturedState.State),
```
Comment (rhrazdil) [inline pkg/nmpolicy/generate.go:96]: isn't `capturedState.State.Raw` []byte already?

Comment (qinqon) [reply pkg/nmpolicy/generate.go:96]: It is but `captureState.State` is not so we have to cast it.

Comment (rhrazdil) [reply pkg/nmpolicy/generate.go:96]: Ah of course

**Review by rhrazdil — APPROVED** [review]
/lgtm
/approve

**Pull request merged**